interview_start_page/src/components/InterviewComponent/CurrentInterviews.tsx

```diff
@@ -62,6 +61,10 @@ const CurrentInterviews: React.FC<Props> = ({ interviewData, setInterviewData })
   }
 
   const addInterview = (companyName: string, deadline: Date) => {
+    if (deadline < new Date()) {
+      return
```

Comment:
Instead of just returning, display an error message underneath the popup like "Please select a date in the future".